python/lsst/daf/butler/core/storageClass.py

```diff
@@ -335,9 +335,29 @@ def __init__(self, config=None):
         self._storageClasses = {}
         self._configs = []
 
+        # Always seed with the default config
+        c = StorageClassConfig()
```

Comment:
In general I think single letter variable names are not good, even in simple cases. If this was named something like defaultConfiguration, there would be no need for the comment. Alternatively, there is little point in naming the output with a single character to just consume it. This could be `self.addFromConfig(StorageClassConfig())` with the comment left above.

Reply:
I'll remove the variable. `defaultConfiguration` seems like a very long variable name for such a short-lived variable.